supertokens_python/recipe/jwt/interfaces.py

```diff
@@ -40,8 +40,9 @@ class CreateJwtResultUnsupportedAlgorithm:
 
 
 class GetJWKSResult:
```

Comment:
double check where all this is used, and you need to pass in a validity_in_secs in all the use of it. It should only be created in the recipe interface.

Reply:
Done